### PR TITLE
Overrides: Include a manual override option to hide something from visualization

### LINKED ISSUE #34781: Plugin "TimeSeries" - deactivate legend for a query
I would like to deactivate the legend for a query in the "TimeSeries" plugin. Unfortunately, this option does not exist. The hints should appear.

grafik

### PR DESCRIPTION
**Pull request by torkelo**
Users should be able to configure some series/fields to be hidden from legend or graph.

Fixes #34781

### CODE REVIEW

**Review by dprokop — COMMENTED** [review]
I think this was a deliberate decision to keep that hidden from overrides, I think it may conflict with an override created by clicking on the legend item. Gonna test it in a bit.

File: packages/grafana-ui/src/options/builder/hideSeries.tsx [hunk 1]
```diff
@@ -43,14 +43,13 @@ export function addHideFrom(builder: FieldConfigEditorBuilder<HideableFieldConfi
     path: 'hideFrom',
     defaultValue: {
       tooltip: false,
-      graph: false,
+      viz: false,
```
Comment (dprokop) [inline packages/grafana-ui/src/options/builder/hideSeries.tsx:46]: There was an issue here @torkelo, otherwise it didn't work for hiding in the vis.

**Review by dprokop — APPROVED** [review]
OK - tested it and seems to work. Couldn't find the issue me and @mckn had when we have implemented this.

File: packages/grafana-ui/src/options/builder/hideSeries.tsx (hunk 1 above)
Comment (torkelo) [reply packages/grafana-ui/src/options/builder/hideSeries.tsx:46]: yea, I noticed that.

Comment (dprokop) [reply packages/grafana-ui/src/options/builder/hideSeries.tsx:46]: Need to dig deeper how to make the default value type safe.

**Comment by dprokop** [comment]
and synced with main to make it green

**Comment by UlrichThiess** [comment]
In Graph (old) i could set in Series override _Legend_ to false. That's not possible in Time Series.

**Pull request merged**